README.md

```diff
@@ -59,6 +60,16 @@ A curated list of awesome Java frameworks, libraries and software.
     - [Websites](#websites)
 - [Contributing](#contributing)
 
+## Alternative Programming Languages
+
+*Languages other than Java that can be used to write JVM applications*
+
+* [Scala](http://www.scala-lang.org/) - Statically typed programming language that fuses the object - oriented model and functional programming ideas.
+* [Groovy](http://www.groovy-lang.org/) - Optionally typed and dynamic language, with static-typing and static compilation capabilities.
+* [Clojure](http://clojure.org/) - Dynamically typed programming language that can be seen as a modern take on Lisp.
+* [Ceylon](http://ceylon-lang.org/) - Statically typed object - oriented language developed by RedHat.
+* [Kotlin](http://kotlinlang.org/) - Statically typed programming language for the JVM, Android and the browser.
```

Comment:
You're right. I simply added a small description found in the language's
website. I'll fix at the first possible moment!
On Aug 17, 2015 11:10 PM, "Andreas Kull" <notifications@github.com> wrote:

> In README.md
> <https://github.com/akullpp/awesome-java/pull/204#discussion_r37230916>:
>
> > @@ -59,6 +60,16 @@ A curated list of awesome Java frameworks, libraries and software.
> >      - [Websites](#websites)
> >  - [Contributing](#contributing)
> >
> > +## Alternative Programming Languages
> > +
> > +*Languages other than Java that can be used to write JVM applications*
> > +
> > +* [Scala](http://www.scala-lang.org/) - Statically typed programming language that fuses the object - oriented model and functional programming ideas.
> > +* [Groovy](http://www.groovy-lang.org/) - Optionally typed and dynamic language, with static-typing and static compilation capabilities.
> > +* [Clojure](http://clojure.org/) - Dynamically typed programming language that can be seen as a modern take on Lisp.
> > +* [Ceylon](http://ceylon-lang.org/) - Statically typed object - oriented language developed by RedHat.
> > +* [Kotlin](http://kotlinlang.org/) - Statically typed programming language for the JVM, Android and the browser.
>
> A bit more information would be nice, not just typing.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/akullpp/awesome-java/pull/204/files#r37230916>.
>